common/djangoapps/student/urls-lms.py

```diff
@@ -0,0 +1,46 @@
+"""
+URLs for student app (lms specific)
+
+IMPORTANT: This is not an appropriate pattern for a common folder.
+TODO: This is a step toward moving lms specific urls/views under the lms.  That
+work still must be completed.
```

Comment:
@nasthagiri @doctoryes @jcdyer @cpennington @efagin: This PR has clarified that we need some discussion here before finalizing.

Note: The earlier approvals on this PR came earlier when all of these endpoints in student/urls-lms.py were in student/urls.py, and there was no change to cms/urls.py.  This would expose all of these endpoints in cms at that point.

The current code clarifies how the different endpoints are currently used in lms and cms.

Here are some outstanding questions:
1. Do we want to expose more of the account related endpoints in cms (i.e. move them to student/urls.py), or is that not a move in the right direction?
2. Is login url something that should be standardized between lms/cms?  For now, is it ok to replicate endpoints in lms/cms that are unused in one or the other?  If not, we'll need a cms app as well.
3. What name do we want to give to the new lms app that should get the lms specific endpoints (urls/views)?  Which endpoints will move with this?

@nasthagiri Feels this shouldn't get merged in a temporary state with lms/cms related files in this common folder.  It would thus be great to make some decisions on this so that we could really start splitting student and land this.